src/transaction/log_page_buffer.c

```diff
@@ -1343,7 +1343,14 @@ logpb_initialize_header (THREAD_ENTRY * thread_p, LOG_HEADER * loghdr, const cha
   loghdr->avg_nlocks = LOG_ESTIMATE_NOBJ_LOCKS;
   loghdr->npages = npages - 1;	/* Hdr pg is stolen */
   loghdr->db_charset = lang_charset ();
-  loghdr->fpageid = 0;
+  if (prm_get_bigint_value (PRM_ID_FIRST_LOG_PAGEID) != 0)
+    {
+      loghdr->fpageid = (LOG_PAGEID) prm_get_bigint_value (PRM_ID_FIRST_LOG_PAGEID);
+    }
+  else
+    {
+      loghdr->fpageid = 0;
+    }
```

Comment:
It looks that we can just do like below without the if-condition:
`loghdr->fpageid = (LOG_PAGEID) prm_get_bigint_value (PRM_ID_FIRST_LOG_PAGEID);` 

Or did you write this to put the case using PRM_ID_FIRST_LOG_PAGEID as an exceptional case explicitly?

Reply:
@wrlawodms I wanted to separate this explicitly, but your suggestion is simpler and nicer. I will reflect.